Resources/skeleton/form/FormType.php.twig

```diff
@@ -39,7 +39,7 @@ class {{ form_class }} extends AbstractType
     /**
      * {@inheritdoc}
      */
-    public function getName()
+    public function getBlockPrefix()
```

Comment:
Can't we just get rid of such method? It makes sense only when you need something different from the entity name (that is not our case)